client/gutenberg/editor/controller.js

```diff
@@ -40,6 +40,13 @@ function getPostID( context ) {
 		return null;
 	}
 
+	if ( 'home' === context.params.post ) {
+		const state = context.store.getState();
+		const siteId = getSelectedSiteId( state );
+
+		return parseInt( getSiteOption( state, siteId, 'page_on_front' ), 10 );
+	}
+
```

Comment:
Don't think this is a blocker, but should this block be copied to `post-editor/controller.js`? Function comment says they're the same but I'm not sure what the difference between `post-editor` and `gutenberg/editor` is, maybe one's legacy?

Reply:
yeah, `post-editor` is for the calypso classic editor. If you go to a route like `wordpress.com/post`, the post-editor controller will redirect it to the gutenberg controller if the user's editor is set to Gutenberg.

Reply:
Does it seem like it's worth updating? Trying to think of things in the future that could catch us out ...
Maybe if the `home` alias got more entrenched then further down the line a user who's opt'd for classic might run into bugs?

Reply:
Maybe, I mean with FSE we do not allow users to opt-out of Gutenberg (FSE doesn't work in any sense with the classic editor). And new users default to Gutenberg anyways. 🤔

I wonder if the Calypso classic editor would work by default. The reason we have to add all these overrides for the Gutenberg back button is because it lives under the iFrame, so we manually override it since Gutenberg has no concept of Calypso's client-side navigation. The post-editor is a normal Calypso page so it might just go back to the right spot by default.